py/cstack.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014 Paul Sokolovsky, 2024 Angus Gratton
```

Comment:
The convention in this project is to put separate authors' copyright on separate lines, ie:
```c
* Copyright (c) 2014 Paul Sokolovsky
* Copyright (c) 2024 Angus Gratton
```